Source/WebCore/dom/CustomElementReactionQueue.cpp

```diff
@@ -99,6 +132,10 @@ class CustomElementReactionQueueItem {
     std::optional<QualifiedName> m_attributeName;
     AtomString m_oldValue;
     AtomString m_newValue;
+
+    std::optional<bool> m_isDisabled;
+    RefPtr<HTMLFormElement> m_associatedForm;
+    AtomString m_state;
```

Comment:
It's a bit wasteful to allocate all these arguments for all callbacks but we can address that afterwards.

Reply:
Added a FIXME in 1df00b9. We could attempt a huge `std::variant` or something.